test/unit_tests/location_service/set_distance_between_user_and_games_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:niira/models/location.dart';
+import 'package:niira/services/location_service.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:niira/utilities/calc_distance.dart';
+import 'package:test/test.dart';
+
+import '../../mocks/data/mock_games.dart';
+
+void main() {
+  test(
+      'locationService.setDistanceBetweenUserAndGames returns list of games with distanceFromUser set',
+      () {
+    // init mock service
+    final geolocator = GeolocatorPlatform.instance;
+    final locationService = LocationService(geolocator);
+    // init mock data
+    final mockUserLocation =
+        Location(latitude: -37.865351, longitude: 144.989012);
+    final mockGames = MockGames().gamesWithoutDistance;
```

Comment:
This is really clear, love it 👍 

Could "gamesWithoutDistance" be static so you don't have to create an instance to call the function?  Or do you generally create a MockGames object to use in objecty type things?

Reply:
good point! i dont really need to do any objecty stuff with it. static should be fine :)